packages/node-cli/src/commands/file/import/ImportFilesInputHandler.js

```diff
@@ -53,7 +53,7 @@ module.exports = class ImportFilesInputHandler extends BaseInputHandler {
 	constructor(options) {
 		super(options);
 		// TODO input handlers shouldn't execute actions. rework this
-		this._sdkExecutor = new SdkExecutor(options.sdkPath);
+		this._sdkExecutor = new SdkExecutor(options.sdkPath, this._executionEnvironmentContext);
```

Comment:
If we add the sdkPath as a property of the BaseInputHandler we can rewrite this line as:
`this._sdkExecutor = new SdkExecutor(this._sdkPath, this._executionEnvironmentContext);` .

Reply:
Applied this change.